docker/android-slave-s2i/Dockerfile

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
Why another folder here? IMHO We should use existing folder and do not produce additional artifacts. 
It's hard to review this now because it's not easy to see what was changed. Having s2i capability doesn't change a lot here. I do not see changes here to justify new file.